Describe mergeColumnFieldValues and mergeRowFieldValues properties (#2143)
DevExpress/devextreme-documentation: api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeRowFieldValues.md

---
---
##### shortDescription
Specifies whether to merge neighbouring cells in the row field if they have the same values.

---

> The following image shows exported pivot grid if you set this property to **true** or leave it unspecified:

Reviewer: ```suggestion
The following images show the exported pivot grid if you set this property to **true** or leave it unspecified
```

> The following image shows exported pivot grid if you set this property to **true** or leave it unspecified...

Reviewer 2: ```suggestion
The following image shows exported pivot grid if you set this property to **true** or leave it unspecified:
```

![DevExtreme PivotGrid: mergeRowFieldValues set to true](/images/UiWidgets/pivotgrid-merge-row-field-values-true.png)

> ... and in this image this property is set to false:

Reviewer 2: ```suggestion
In the image below, the **mergeRowFieldValues** property is set to **false**:
```

> In the image below, the **mergeRowFieldValues** property is set to **false**:

Reviewer: ```suggestion
or set the **mergeRowFieldValues** property to **false**:
```
